src/Esfa.Vacancy.Domain/Entities/VacancySearchParameters.cs

```diff
@@ -18,6 +18,7 @@ public class VacancySearchParameters
         public int? DistanceInMiles { get; set; }
         public bool HasGeoSearchFields => Latitude.HasValue && Longitude.HasValue && DistanceInMiles.HasValue;
         public SortBy SortBy { get; set; }
+        public int? ProviderUkprn { get; set; }
```

Comment:
This should be a long, across DAS we use long for UKPRN. If this is a new field being added to the index, I would suggest long to keep in line for how we are using it in the v2 world aligned with DAS.

Reply:
👍 Yep. I'll change this. I let v1 direct the int but I'll change to a long.